src/python/friendzone/nwx2molssi/nwchem_via_molssi.py

```diff
@@ -114,18 +115,18 @@ def load_qcengine_modules(mm):
 
     The final set of modules is the Cartesian product of all of the above.
 
+    This function is a no-op if NWChem is not installed.
+
     :param mm: The ModuleManager that the NWChem Modules will be loaded into.
     :type mm: pluginplay.ModuleManager
```

Comment:
Does Sphinx know about docstrings for the python bindings? If so, you could link to them here by changing this to `:class:'pluginplay.ModuleManager'` (replace the `'` with back ticks, I couldn't figure out how to stop code formatting issues otherwise) if that isn't already done automatically on `:type :` directives.